### PR TITLE
iso hotswap: remove obsolete patch in qemu and fix xl patch

### PR DESCRIPTION
**Pull request by mahantesh-ais**
OXT-777

Signed-off-by: Mahantesh Salimath <mahantesh.openxt@gmail.com>

### CODE REVIEW

File: recipes-openxt/qemu-dm/qemu-dm.inc [hunk 1]
```diff
@@ -25,7 +25,6 @@ SRC_URI += "file://compile-time-stubdom-flag.patch \
             file://surfman-dcl.patch \
             file://audio-policy.patch \
             file://msix-cap-disable.patch \
-            file://xenstore-based-ISO-media-change-detection-for-both-s.patch \
```
Comment (dpsmith) [inline recipes-openxt/qemu-dm/qemu-dm.inc]: If you are dropping the patch queue, might I suggest removing the patch so there is no confusion about carrying the patch forward on the next uprev of qemu.

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt/builds/1263
@mahantesh-ais, please add a commit to remove the old patch.

File: recipes-openxt/qemu-dm/qemu-dm.inc (hunk 1 above)
Comment (mahantesh-ais) [reply recipes-openxt/qemu-dm/qemu-dm.inc]: Ack, fixed.

**Comment by eric-ch** [comment]
Building: [1266](http://openxt-builder.ainfosec.com:8010/builders/openxt/builds/1266)

**Comment by eric-ch** [comment]
Tested with in a Windows 10 64 bits HVM guest with PV tools. ISO change through the UI worked fine.

LGTM, will merge soon.

**Pull request merged**